src/debugger/symbols.c

```diff
@@ -97,6 +97,13 @@ void mDebuggerLoadARMIPSSymbols(struct mDebuggerSymbols* st, struct VFile* vf) {
 			continue;
 		}
 
+        char* buf2;
+
+        if ((buf2 = strchr(buf, ',')) != NULL) {
+            // Commas separate names from function sizes
+            *buf2 = '\0';
+        }
```

Comment:
Tabs, not spaces. Also, move the assignment to the variable declaration. There's no reason it needs to be inside the `if`, so it shouldn't be there.